airflow/kubernetes_executor_templates/dags_in_image_template.yaml

```diff
@@ -0,0 +1,84 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Instead of creating a new top-level directory, I think it is better to have a `example_templates` folder under `airflow/kubernetes` folder